sql-to-dbsp-compiler/lib/json/src/lib.rs

```diff
@@ -0,0 +1,190 @@
+#![deny(missing_docs)]
+
+//! Functions related to `JSON` support in DBSP
+
+/// The actual Json type.
+pub type Json = ijson::IValue;
+
+/// Parses the given string to [`Json`].
+///
+/// # Panics
+/// - Panics if it fails to deserialize to a valid Json.
+pub fn parse_json(value: impl AsRef<str>) -> Json {
+    serde_json::from_str(value.as_ref()).unwrap_or_else(|_| {
+        panic!(
+            "cannot deserialize given string to JSON: {}",
```

Comment:
the json string may be very long, you don't want to print all of it. Maybe you can create a utility function (if we don't have one) which truncates it to a reasonable size

Reply:
I think we can just take the first 100 (or the max, if max < 100) characters of the string.